src/main/java/uk/nhs/hee/tis/revalidation/connection/repository/DiscrepanciesElasticSearchRepository.java

```diff
@@ -21,11 +21,23 @@
 
 package uk.nhs.hee.tis.revalidation.connection.repository;
 
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.elasticsearch.annotations.Query;
 import org.springframework.data.elasticsearch.repository.ElasticsearchRepository;
 import org.springframework.stereotype.Repository;
 import uk.nhs.hee.tis.revalidation.connection.entity.DiscrepanciesView;
+import uk.nhs.hee.tis.revalidation.connection.entity.ExceptionView;
 
 @Repository
 public interface DiscrepanciesElasticSearchRepository
     extends ElasticsearchRepository<DiscrepanciesView, String> {
+
+  @Query("{\"bool\":{\"must_not\":{\"match\":{\"membershipType\":\"MILITARY\"}},"
```

Comment:
Could it be better to move this filter to the query for populating `discrepancies` alias?